huaweicloud/services/dbss/resource_huaweicloud_dbss_instance.go

```diff
@@ -156,6 +161,13 @@ func ResourceInstance() *schema.Resource {
 				ForceNew:    true,
 				Description: `Specifies the IP address.`,
 			},
+			"action": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"start", "stop", "reboot",
+				}, true),
```

Comment:
```suggestion
				}, false),
```
The code uses the value of this field to concatenate the URL, so the case cannot be ignored.